flink-streaming-java/src/main/java/org/apache/flink/streaming/api/operators/AbstractStreamOperator.java

```diff
@@ -382,6 +382,11 @@ public void notifyCheckpointAborted(long checkpointId) throws Exception {
         stateHandler.notifyCheckpointAborted(checkpointId);
     }
 
+    @Override
+    public void notifyCheckpointSubsumed(long checkpointId) throws Exception {
+        stateHandler.notifyCheckpointSubsumed(checkpointId);
+    }
```

Comment:
There is also `AbstractStreamOperatorV2`...

Reply:
I see, thanks for your reminder.